core/trino-main/src/main/java/io/trino/execution/scheduler/faulttolerant/ByEagerParentOutputStatsEstimator.java

```diff
@@ -49,6 +49,6 @@ public Optional<OutputDataSizeEstimateResult> getEstimatedOutputDataSize(StageEx
         for (int i = 0; i < outputPartitionsCount; ++i) {
             estimateBuilder.add(0);
         }
-        return Optional.of(new OutputDataSizeEstimateResult(estimateBuilder.build(), OutputDataSizeEstimateStatus.ESTIMATED_FOR_EAGER_PARENT));
+        return Optional.of(new OutputStatsEstimateResult(estimateBuilder.build(), 0, OutputStatsEstimateStatus.ESTIMATED_FOR_EAGER_PARENT));
```

Comment:
in the other place where 0 was  we have this comment

```
// TODO: For now we can skip calculating outputRowCountEstimate since we won't run adaptive planner in the case of small inputs
```

style aside, is 0 a safe value? 

cc @losipiuk

Reply:
It actually is fine I think. The code currently assumes (and it is even conveyed by the comment above) that this estimator is used solely for stages which would produce small output.
Zero is not exactly strict of course - but is in correct ballpark.

Commnt above:
```
        // use empty estimate as fallback for eager parents. It matches current logic of assessing if node should be processed eagerly or not.
        // Currently, we use eager task exectuion only for stages with small FINAL LIMIT which implies small input from child stages (child stages will
        // enforce small input via PARTIAL LIMIT)
```